eng/pipelines/templates/steps/build-artifacts.yml

```diff
@@ -24,11 +24,11 @@ steps:
       PackageName: "azure-template"
       ServiceDirectory: "template"
       TestPipeline: ${{ parameters.TestPipeline }}
-
+  - template: /eng/common/pipelines/templates/steps/set-default-branch.yml
   - script: |
       echo "##vso[build.addbuildtag]Scheduled"
     displayName: 'Tag scheduled builds'
-    condition: and(eq(variables['Build.SourceBranchName'],'master'),eq(variables['Build.Reason'],'Schedule'))
+    condition: and(eq(variables['Build.SourceBranchName'], variables['DefaultBranch']), eq(variables['Build.Reason'],'Schedule'))
```

Comment:
![image](https://user-images.githubusercontent.com/48036328/114955095-bc416600-9e10-11eb-9920-8d8911b24763.png)

Tested in core pipeline: https://dev.azure.com/azure-sdk/public/_build/results?buildId=845791&view=logs&j=3dc8fd7e-4368-5a92-293e-d53cefc8c4b3&t=3c663f50-2031-5e5a-5a27-a536dda269a8